src/core/ui.js

```diff
@@ -24,6 +24,14 @@ styleElement.classList.add("removeOnSave");
 
 document.head.appendChild(styleElement);
 
+let body = $('body');
```

Comment:
As this is github related, the best place for this code would be "src/github.js".

Reply:
this should have been "src/core/github.js"

Reply:
Got it